peer/main.go

```diff
@@ -244,15 +244,22 @@ func main() {
 	flags := nodeStartCmd.Flags()
 	flags.BoolVarP(&chaincodeDevMode, "peer-chaincodedev", "", false, "Whether peer in chaincode development mode")
 
+	var alternativeCfgPath = os.Getenv("PEER_CFG_PATH")
+	if alternativeCfgPath != "" {
```

Comment:
@muralisrini When I added https://github.com/hyperledger/fabric/issues/1967, I removed the behave test of noop that used PEER_CFG_PATH (in order to load a custom config with noop syscc enabled, but this functionality was removed in master, see the comment history). If the whitelist functionality is OK, I think I can mark the issue as solved and readd the noop bdd test. What is your opinion? If you agree, I readd the test (where PEER_CFG_PATH will be set and a custom config will be used with noop enabled) and the PR can be merged.